src/@openeventkit/event-site/templates/marketing-page-template/styles.module.scss

```diff
@@ -0,0 +1,16 @@
+.mainColumn {
+}
+
+.masonry {
+  overflow: hidden;
+  .grid {
+    display: flex;
+  }
+  .slider {
+    margin-bottom: -5px;
+  }
+  .masonryImage { // hack for white border/lines between masonry images
+    width: 101%;
```

Comment:
This will cause images to overlap, but its imperceptible with currently used image on themes masonry.